examples/misc/lib/effective_dart/style_bad.dart

```diff
@@ -10,3 +10,10 @@ class Dice {
   static final NUMBER_GENERATOR = new Random();
 }
 // #enddocregion const-names
+
+oneLineIf(dynamic overflowChars, dynamic other) {
```

Comment:
I usually don't name the function, given that the `docregion` name is what is used.

Reply:
This is a top-level function, so it needs a name.